UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API.csproj

```diff
@@ -262,5 +262,9 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Data\FileShareService\ENC\FR\" />
+  </ItemGroup>
+
```

Comment:
Do we need it ? If not please remove it.

All files under newly added countries should have **CopyToOutputDirectory** property as **Copy if newer**

Reply:
Updated. Thanks